src/logging.gleam

```
@@ -33,3 +35,19 @@ pub fn log(level: LogLevel, message: String) -> Nil {
 
 @external(erlang, "logger", "log")
 fn erlang_log(level: LogLevel, message: String) -> DoNotLeak
+
+@external(erlang, "logging_ffi", "set_primary_config")
+fn set_primary_config_level(
+  key: Key,
+  level: LogLevel,
+) -> Result(Nil, dynamic.Dynamic)
+
+/// Change the log visibility level to be output from the default of `Info`.
+///
+pub fn set_level(level: LogLevel) -> Result(Nil, dynamic.Dynamic) {
```

Please revert this to the previous API. I don't want to expose the error value at all, rather silently do nothing in that case like the existing code does.

---

Oh ok! fair enough, will do!